### PR TITLE
Feature/ma 51 auth manager initial checkin

### PR DESCRIPTION
**Pull request by liujoshua**
Gets mPower compiling with code changes introduced by auth manager check-in. New code is largely unexercised so there should be no behavior change

### CODE REVIEW

**Comment by dephillipsmichael** [comment]
I'm feel like Travis is more of a bottleneck for mPower, especially with abortOnError = false.  I'd vote to remove it.

File: MPower/app/build.gradle [hunk 1]
```diff
@@ -26,6 +26,10 @@ android {
     packagingOptions {
         exclude 'META-INF/LICENSE'
     }
+
+    lintOptions {
```
Comment (dephillipsmichael) [inline MPower/app/build.gradle]: remove this now as well

**Comment by dephillipsmichael** [comment]
Remove the lint on error, then you are free to merge

File: MPower/app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerDataProvider.java [hunk 2]
```diff
@@ -18,8 +19,8 @@
 
     public MPowerDataProvider(Context context) {
         // TODO give path to permission file for uploads
-        super(BRIDGE_URL, STUDY_ID, USER_AGENT,
-                new MPowerResourceManager.PemResource(CERTIFICATE_NAME));
+        super(BridgeManagerProvider.getInstance(context), BRIDGE_URL, STUDY_ID, USER_AGENT,
```
Comment (liujoshua) [inline MPower/app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerDataProvider.java]: This change is about to be reverted

**Pull request merged**